scripts/preview-gh-pages.js

```diff
@@ -6,23 +6,25 @@ const { createServer } = require('http-server');
 const portfinder = require('portfinder');
 portfinder.basePort = 4000;
 
-(async() => {
-  if (!fs.existsSync('docs/dist/seek-style-guide')) { // eslint-disable-line no-sync
+(async () => {
+  if (!fs.existsSync('docs/dist/seek-style-guide')) {
+    // eslint-disable-line no-sync
```

Comment:
This ignore needs to be addressed

Reply:
Done. It's interesting that `scripts/` isn't currently being linted by the **lint** step, might be good to change that.